ingestion/src/metadata/data_quality/interface/sqlalchemy/sqa_test_suite_interface.py

```diff
@@ -166,13 +165,27 @@ def run_test_case(
                 test_case.testDefinition.fullyQualifiedName,
             )
 
+            runtime_parameter_setter = TestHandler.runtime_parameter_setter
```

Comment:
@sushi30 can you add a comment here. I forgot how it works and it might help us in the future.